src/h3lib/lib/h3Index.c

```diff
@@ -355,18 +356,18 @@ int H3_EXPORT(compact)(const H3Index* h3Set, H3Index* compactedSet,
             }
         }
         // Determine which parent hexagons have a complete set
-        // of children and put them in the compactableHexes array
-        int compactableCount = 0;
+        // of children and put them in the compactCellsableHexes array
+        int compactCellsableCount = 0;
```

Comment:
This one looks like a find&replace error

Reply:
That it is. Will fix.